src_c/line.c

```diff
@@ -1,94 +1,31 @@
 #include "include/pygame.h"
```

Comment:
Could you change these? the actual file name is "Include" not "include", it's driving my IDE crazy and i don't think it's a good thing. We either rename the "Include" file to "include" or change each `#include` in the .c files